tests/common.go

```diff
@@ -6649,6 +6652,42 @@ func IsGCPBucketEmpty(bucketName string) (bool, error) {
 	return false, nil
 }
 
+// IsAzureBlobEmpty returns true if bucket empty else false
+func IsAzureBlobEmpty(containerName string) (bool, error) {
+	_, _, _, _, accountName, accountKey := GetAzureCredsFromEnv()
+	azureRegion := os.Getenv("AZURE_ENDPOINT")
+	urlStr := fmt.Sprintf("https://%s.blob.core.windows.net/%s", accountName, containerName)
+	if azureRegion == "CHINA" {
```

Comment:
can we add this into constant "CHINA"

Reply:
@sn-px Please advise, as it is just the name of the ENV variable which is passed from Jenkins file.

Reply:
@sn-px Addressed.